recipe/meta.yaml

```diff
@@ -25,14 +24,13 @@ requirements:
   build:
     - python  # [win]
     - vc   9   # [win and py27]
-    - vc  10   # [win and py34]
-    - vc  14   # [win and py35]
+    - vc  14   # [win and py >= 35]
     - postgresql  {{ version }}
   run:
     - vc   9   # [win and py27]
-    - vc  10   # [win and py34]
-    - vc  14   # [win and py35]
+    - vc  14   # [win and py >= 35]
     - openssl 1.0.*
+    - krb5
```

Comment:
Since this is basically pulling out libpq from a full postgresql install and repackaging it we still need it at runtime since we dont depend on postgresql here.